ccvm_simulators/ccvmplotlib/ccvmplotlib.py

```diff
@@ -319,3 +319,27 @@ def apply_default_succ_prob_styling(
 
         # call tight layout
         fig.tight_layout()
+
+    # TODO: This will be done in issue #137. Currently, just for testing purposes
+    @staticmethod
+    def plot_ETS(
+        metadata_filepath: str,
+        problem: str,
+        energy_max_func: callable,
+    ):
+        """Plot a problem-specific Energy-To-Solution metadata solved by a CCVM
+        solver.
+
+        Args:
+            metadata_filepath (str): A file path to metadata.
+            problem (str): A problem type.
+            energy_max_func (callable): A function that returns the maximum energy
+                consumption of a given problem size..
+        """
+        problem_metadata = ProblemMetadataFactory.create_problem_metadata(
+            problem, "wallclock"
+        )
```

Comment:
I think the decision was to remove the variable `TTS_Type` and  its values `wallclock` or `physical` with the parameter `machine` and its corresponding values. It seems like `TTS_Type` is still used in the metadata files.
I'm not sure if @ChanWooY-1QBit is removing these in his PR for updating TTS or not, but they are not separate.

Reply:
Done. This has been fixed in the main code.